### ISSUE TITLE
Extensions invalid directory

### ISSUE THREAD

**Issue by Lou511** [obsidianforensics/hindsight#41]
Hi Ryan.  I'm working with data from a Windows machine, and I'm getting the following traceback error:

Traceback (most recent call last):
  File "site-packages\bottle.py", line 862, in _handle
  File "site-packages\bottle.py", line 1732, in wrapper
  File "hindsight_gui.py", line 163, in do_run
  File "pyhindsight\analysis.py", line 438, in run
  File "pyhindsight\browsers\chrome.py", line 1852, in process
  File "pyhindsight\browsers\chrome.py", line 827, in get_extensions
WindowsError: [Error 267] The directory name is invalid: 'W:\\Parent_directory\\User Data\\Default\\Extensions\\mallpejgeafdahhflmliiahjdpgbegpk/*.*'

Can you help with this error?  Or if this error is not fatal to results-output, is there a way for Hindsight to exit despite the error, so I can get the parsed results?  When I look at the CLI, I see that a bunch of records were parsed.

**Comment by obsidianforensics** [comment]
Hey there - sorry you're having this problem. That seems like a strange error, since Hindsight is just trying to list a directory there, nothing complicated. Can you tell me what the directory structure looks like for your input data? 

Regardless, there should be some better error checking there in Hindsight, I'll add that in.

If you just want a really fast workaround (and Extension version info isn't critical to your case, since that's all that part does), you can delete/rename the 'Extensions' directory in the input data to something different (Extensions2 for example) and Hindsight will not process it.

**Comment by Lou511** [comment]
Hi Ryan.  Thanks for the workaround.  I just made the modification, and everything seems to be going fine.  The input data directory structure..... can you rephrase this?  Do you mean, a listing of the folders inside the "Default" directory?

**Comment by obsidianforensics** [comment]
Yes, if you are on Windows, something like the output of 'dir /s'
